embedded-kafka/src/main/java/com/playtika/test/kafka/configuration/KafkaContainerConfiguration.java

```diff
@@ -212,4 +228,48 @@ public KafkaTopicsConfigurer kafkaConfigurer(
         return new KafkaTopicsConfigurer(kafka, zookeeperProperties, kafkaProperties);
     }
 
-}
\ No newline at end of file
+    /**
+     * Create folder (and parent folder if necessary) with write permissions for current user
+     *
+     * @param path folder to create
+     */
+    private void createPathAndParentOrMakeWritable(Path path) {
+        Stream.of(path.getParent(), path).forEach(p -> {
+            if (p.toFile().isDirectory()) {
+                makeWritable(p);
+            } else {
+                try {
+                    log.info("Create writable folder: {}", p);
+                    Files.createDirectory(p, PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rwxrwxrwx")));
+                } catch (FileAlreadyExistsException e) {
+                    makeWritable(p);
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+            }
+        });
+    }
+
+    /**
+     * Make folder writable
+     *
+     * @param path folder to make writable
+     */
+    private void makeWritable(Path path) {
+        PosixFileAttributeView fileAttributeView = Files.getFileAttributeView(path, PosixFileAttributeView.class);
+        if (fileAttributeView == null) {
+            log.warn("Couldn't get file permissions: {}", path);
+            return;
```

Comment:
If we couldnt make the directory writable -- will the container be usable? if no than we should thrown an exception

Reply:
The method allows `null`, but it is unlikely, I think return is enough in that extreme case.
Do you want me to change it nevertheless?

Reply:
I mean that when the method returns null, you log `Couldn't get file permissions:`. Does it mean that the directory will not be writable? If yes, than we definitely need to throw an exception